### ISSUE TITLE
Teams with 2 language packs need a second download button or notice

### ISSUE THREAD

**Issue by timse201** [WordPress/wporg-main-2022#515]
The Dutch, German and Swiss teams are maintaining a second language package. As they only differ in the form of the address (formal/informal), we don't provide a second landing page. Therefore, we (+ the download page team) need a discussion about how we handle this and what is possible.

1. solution
Please add a second download button

2. solution
Add a notice that a second language pack is available for users to choose from within WordPress

@swissspidy @jeroenrotty @tacoverdo @remkus @ocean90 @Zodiac1978 @nickw108

**Comment by timse201** [comment]
> https://github.com/WordPress/wporg-mu-plugins/issues/669#issuecomment-2480055841
> There are a couple more: Serbian and Portuguese (Portugal) have variants for various spelling conventions.
> Technically, there are a couple of variants on Catalan, but at least Valenciano has been defined separately, too.

_Originally posted by @tobifjellner in https://github.com/WordPress/wporg-mu-plugins/issues/669#issuecomment-2480055841_

**Comment by nickweisser** [comment]
I think solution 2 would be enough. Maybe include a screenshot where users can change it in WordPress admin.

**Comment by timse201** [comment]
I prefer a second download button because it is more user-friendly.

**Comment by ocean90** [comment]
A second download button doesn't make sense since there would be nothing to download.

**Comment by Zodiac1978** [comment]
So the question is: Would it be possible to detect that a locale has variants to make this additional info conditional?

I think it could be confusing to add this info but the locale has no variants.

**Comment by remkus** [comment]
I agree on adding the extra locale version as an extra button. Very simple solution, easily implemented.

**Comment by timse201** [comment]
A release package was built and we can download our second package e.g.:
https://de.wordpress.org/wordpress-6.7.1-de_DE_formal.zip

so it seems it is only a design question.

**Comment by timse201** [comment]
@pedro-mendonca @alvarogois can you please join aus. Is it needed for pt_pt_ao90?

**Comment by alvarogois** [comment]
@timse201 we have a very specific (and tricky) context in Portugal. Forgive me for the long comment but is just for the sake of context. You can skip to the last paragraph as the following two are just for context.

An orthography reform was implemented in the last decades in Portugal and Portuguese speaking countries (shrouded in considerable controversy, and still not fully adopted). About 10 years ago there was a vote in the local community on whether the WordPress default orthography for Portuguese (Portugal) should be updated or not. And at the time [it was decided (there was a community vote) to maintain the old orthography](https://wp-portugal.com/2013/05/29/wordpress-em-portugues-sem-acordo-ortografico/). For two reasons: it is possible to automatically convert the old orthography to the new one, but the reverse is not possible. In other words, with just one translation we can have both, although this requires (in addition to the conversion process outside of GlotPress, done manually by the translation team) the user, then, to choose the AO90 variant to have access to the new spelling. The second reason that causes indecision in adopting the new orthography is the fact that, when changing the default WordPress pt_PT language, all sites would be automatically updated to the new orthography, without decision-making power for the respective site managers. Naturally, this places a huge responsibility on whoever makes the decision to change the standard spelling.

From the moment the variants in GlotPress came into existence, and we were able to have an AO90 variant, some members of the Community organized themselves to create a [plugin](https://wp-portugal.com/traducao/plugin-pt-ao90/) that would allow not only to change the orthography but also maintain a fallback to the old orthography, in order to make available most translations in plugins and themes and without the option being simply English as the default fallback.

**TL;DR**

Regarding the options discussed here, I would say that **both options are a step forward**. And it wouldn't even shock me if both were adopted simultaneously. A second button would be helpful for whom downloads WordPress from the website (for example, first timers), the second could be available also when a pt_PT WordPress install is updated, showing there is another orthography available. Furthermore, this would give the translation team more security when, inevitably, the new orthography variant is chosen as the official default orthography, as it would be clear to anyone who wanted to keep the old orthography that this would still be possible.

**Comment by timse201** [comment]
Interesting. Thank you for your explanation. You're right.

So the optimal solution for Rosetta could be that there is a second download button and to give a hint that users can add fallback variant(s) in the settings? swissspidy is currently working on integrating this feature into Core.

**Comment by alvarogois** [comment]
Yes, I would say so. If those options are in fact available, no reason to not show them to the potential users. It would improve clear information about available variants and probably adoption.

**Comment by timse201** [comment]
I have taken up the wishes and created 2 drafts.

I have taken up @nickw108's wish to include a screenshot and would like to link to Playground here so that the screenshot does not have to be updated. The link can be customized and lead to the corresponding language. e.g.:
https://playground.wordpress.net/?login=1&url=%2Fwp-admin%2Foptions-general.php&language=de_DE

We would then have 2 download buttons we can translate as required (default -> du/Sie etc.).

1. draft
![Image](https://github.com/user-attachments/assets/570b9264-0a58-40cd-8f3e-6efea3ba9719)

2. draft
This addition is a bit long. So I decided to make a second draft to keep the simplicity and shortness of this page. According to issue #521, it could be that the javascript popup that appears after clicking download moves to php. There could be an addition informing about the site language picker.

![Image](https://github.com/user-attachments/assets/1989e40b-40a5-4b0a-9bc6-6cff87022071)
![Image](https://github.com/user-attachments/assets/3aee1604-abc2-4d85-9379-3d6d91503b68)

(The ticket was placed on needs design feedback)

**Comment by chrislaai** [comment]
Button styling looks wrong in responsive mode-> https://wordpress.org/download/
![Image](https://github.com/user-attachments/assets/d12f8ad4-24a6-4ccb-9a03-b3cda34c1f1e)

**Comment by Zodiac1978** [comment]
@chrislaai This is correct, but has nothing to do with this issue about adding a second button for locale variants (like formal language in DE or NL).

And this is not the only problem. Looking at this video I made there are other problems as well, like bad breaking words, orphan letters, unnecessary margins, etc.
![Image](https://github.com/user-attachments/assets/a87f6183-dd88-40d6-9ae7-57f2445e2206)

**Comment by ryelle** [comment]
@chrislaai @Zodiac1978 This was probably introduced when the Download button was updated (https://github.com/WordPress/wporg-main-2022/pull/531, https://github.com/WordPress/wporg-mu-plugins/pull/675). I opened #535 with your reports.
